src/main/resources/application-prod.yaml

```diff
@@ -76,6 +76,7 @@ rolle:
 
 YRKESSKADE_DOKGEN_API_URL: https://yrkesskade-dokgen.intern.nav.no
 YRKESSKADE_KODEVERK_API_URL: https://yrkesskade-kodeverk.intern.nav.no
+YRKESSKADE_JSON_TO_PDF_API_URL: https://yrkesskade-json-to-pdf.intern.nav.no
```

Comment:
Jeg lurer på om vi kan gå over til å samle integrasjoner i api.client.... som på linje 81 - vi kan godt ta inn verdiene som parametere, men det gjør det litt mer oversiktlig

Reply:
Jeg synes det er penere, så kan godt gjøre det. Gjorde det slik for å gjøre det likt som det som var der fra før